ballerina/tests/mock_service.bal

```diff
@@ -0,0 +1,283 @@
+                "userCount": 2,
+                "userCountDescription": "users",
+                "hasUnlimitedSeats": false,
+                "platform": false
+            }
+        ];
+    }
+
+     // Mock endpoint to retrieve a specific ApplicationRole based on key
+    resource function get rest/api/'3/applicationrole/[string 'key]() returns ApplicationRole|http:Unauthorized|http:Forbidden|http:NotFound {
+        return 
+        { 
+            "key": "jira-software",
+            "name": "Jira Software"
+        }; 
+    }
+
+    // Mock endpoint for auditing records
+    resource function get rest/api/'3/auditing/'record(string? filter, string? 'from, string? to, int:Signed32 offset = 0, int:Signed32 'limit = 1000) returns AuditRecords|ErrorCollectionUnauthorized|ErrorCollection{
+        // Mock response with a sample record for testing
+        return
+        {
+            "records": [
+                {
+                    "id": 1001,
+                    "summary": "User logged in",
+                    "created": "2023-10-25T12:34:56.000Z",
+                    "category": "authentication"
+                }
+            ]
+        };
+    }
+
+    resource function get rest/api/'3/classification\-levels(("PUBLISHED"|"ARCHIVED"|"DRAFT")[]? status, "rank"|"-rank"|"+rank"? orderBy) returns DataClassificationLevelsBean|http:Unauthorized {
+        // Mock response mimicking a successful response with `classifications` key
+     return 
+        {
+            "classifications": []
+        };
+    }
+
+    resource function get rest/api/'3/dashboard("my"|"favourite"? filter, int:Signed32 startAt = 0, int:Signed32 maxResults = 20) returns PageOfDashboards|ErrorCollection|ErrorCollectionUnauthorized {
+        // Mock response mimicking the structure of PageOfDashboards
+        return 
+        {
+            "dashboards": []
+        };
+    }
+
+    resource function post rest/api/'3/dashboard(@http:Payload DashboardDetails payload, boolean extendAdminPermissions = false) returns Dashboard|ErrorCollection|ErrorCollectionUnauthorized {
+        // Mock response body mimicking the actual API response
+       return 
+        {
+            "name": "Auditors dashboard",
+            "id":"10001",
+            "description": "A dashboard to help auditors identify sample of issues to check.",
+            "sharePermissions": [],
+            "editPermissions": []
+        };
+    }
+
+    resource function get rest/api/'3/filter/defaultShareScope() returns DefaultShareScope|http:Unauthorized {
+        // Mock response with a default share scope
+        return 
+        {
+            "scope": "GLOBAL"
+        };
+    }
+
+    resource function put rest/api/'3/filter/defaultShareScope(@http:Payload DefaultShareScope payload) returns DefaultShareScope|http:BadRequest|http:Unauthorized {
+
+        // Mock response, echoing the request scope value
+        return 
+        {
+            "scope":"AUTHENTICATED"
+        };
```

Comment:
Still there are formatting issues. You can format the code using vscode if it is hard to do manually. (select the section -> right click -> format)